src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,412 @@
+                <StatusMenuItem eventKey="draft" context={context}>Private</StatusMenuItem>
+                <Dropdown.Divider />
+                {/* These statuses currently not available.
+                <StatusMenuItem active={context.status === "archived to project"} eventKey="archived to project">Archive to Project</StatusMenuItem>
+                <StatusMenuItem active={context.status === "archived"} eventKey="archived">Archive to Lab</StatusMenuItem>
+                */}
+                <StatusMenuItem eventKey="deleted" context={context}>Delete</StatusMenuItem>
+            </DropdownButton>
+        );
+    }
+
+    saveButton(){
+        const { session, context } = this.props;
+        const { saveLoading } = this.state;
+        const tooltip = "Save the current view shown below to this display";
+
+        const editPermission  = this.havePermissionToEdit();
+
+        return (
+            <button type="button" onClick={this.handleSave} disabled={!editPermission || saveLoading} className="btn btn-success" key="savebtn" data-tip={tooltip}>
+                <i className={"icon icon-fw icon-" + (saveLoading ? 'circle-notch icon-spin fas' : 'save fas')}/>&nbsp; Save
+            </button>
+        );
+    }
+
+    cloneButton(){
+        const { session } = this.props;
+        const { cloneLoading } = this.state;
+        const tooltip = "Create your own new HiGlass Display based off of this one";
+
+        return (
+            <button type="button" onClick={this.handleClone} disabled={!session || cloneLoading} className="btn btn-success" key="clonebtn" data-tip={tooltip}>
+                <i className={"icon icon-fw icon-" + (cloneLoading ? 'circle-notch icon-spin fas' : 'clone far')}/>&nbsp; Clone
+            </button>
+        );
+    }
+
+    /**
+     * Is used to call {function} `props.toggleFullScreen` which is passed down from app.js BodyElement.
+     */
+    handleFullscreenToggle(){
+        const { isFullscreen, toggleFullScreen } = this.props;
+        setTimeout(toggleFullScreen, 0, !isFullscreen);
+    }
+
+    fullscreenButton(){
+        const { isFullscreen, toggleFullScreen } = this.props;
+        if(typeof isFullscreen === 'boolean' && typeof toggleFullScreen === 'function'){
+            return (
+                <button type="button" className="btn btn-outline-dark" onClick={this.handleFullscreenToggle} data-tip={!isFullscreen ? 'Expand to full screen' : null}>
+                    <i className={"icon icon-fw fas icon-" + (!isFullscreen ? 'expand' : 'compress')}/>
+                </button>
+            );
+        }
+        return null;
+    }
+
+    onSaveMicroscope(microscope, complete) {
+        // // Do some stuff... show pane for people to browse/select schema.. etc.
+        // setTimeout(function() {
+        //     console.log(microscope);
+        //     complete(microscope.Name);
+        // });
+
+        const { context, href } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['context' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)